### PR TITLE
fix: fix asset mutation when page_resolutions_array is None

### PR DESCRIPTION
**Pull request by Jonas1312**

### CODE REVIEW

File: src/kili/entrypoints/mutations/asset/helpers.py [hunk 1]
```diff
@@ -38,9 +38,10 @@ def process_update_properties_in_assets_parameters(properties) -> Dict:
         map(is_none_or_empty, properties["to_be_labeled_by_array"])
     )
 
-    properties["page_resolutions_array"] = _ensure_page_resolution_dicts(
-        properties["page_resolutions_array"]
-    )
+    if "page_resolutions_array" in properties:
```
Comment (PierreLeveau) [inline src/kili/entrypoints/mutations/asset/helpers.py:41]: "page_resolutions_array" is always in properties

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

File: src/kili/entrypoints/mutations/asset/helpers.py (hunk 1 above)
Comment (Jonas1312) [reply src/kili/entrypoints/mutations/asset/helpers.py:41]: not for `change_asset_external_ids`

Comment (PierreLeveau) [reply src/kili/entrypoints/mutations/asset/helpers.py:41]: indeed

Comment (PierreLeveau) [reply src/kili/entrypoints/mutations/asset/helpers.py]: merging the new branch that will make this change not necessary

**Pull request merged**